toolbox/incari/object/get-scale.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+![The Get Scale Node.](../../../.gitbook/assets/toolbox/incari/object/get-scale.PNG)
+
+| Input | Output |
+| :--- | :--- |
+| Object ID | Scale |
+
+
+**Get Scale** returns the *scale* of an **Object** in a **Scene**. The **Object ID** is given as input to the **Node**.
+
```

Comment:
# Attributes

|Attribute|Type|Description|
|---|---|---|
|`Object`|**ObjectID**|The **Object** that you wish to return the `scale` of, if one isn't provided in the  `Object ID` **Socket**.|